test/jit/test_class_type.py

```diff
@@ -1180,6 +1180,15 @@ def attr(self) -> int:
             def unsupported(self) -> int:
                 return sum([self.a])
 
+            @torch.jit.unused
```

Comment:
The convention of `torch.jit.unused` is that the function can occur as a call within a script'd. function but will error if used.  Does. that work here ?

Reply:
If a function tries to call this, an error will be produced that says that there is no such attribute. Not quite as helpful as the usual `unused` error, but the behaviour is closed to `unused` than to `ignore`.

Reply:
Okay, sgtm.